conf/messages

```diff
@@ -493,6 +493,60 @@ otherBeneficiary.whenRemoved.error.invalid = Enter a real date when the benefici
 otherBeneficiary.whenRemoved.error.future = The date the beneficiary was removed from the trust must be today or in the past
 otherBeneficiary.whenRemoved.error.past = The date the beneficiary was removed must be on or after the date they were added
 
+otherBeneficiary.description.title = What is the description for the class of beneficiaries?
+otherBeneficiary.description.heading = What is the description for the class of beneficiaries?
+otherBeneficiary.description.checkYourAnswersLabel = What is the description for the class of beneficiaries?
+otherBeneficiary.description.hint = This must be 56 characters or less. For example, future grandchildren and future great-grandchildren.
+otherBeneficiary.description.error.required = Enter the description for the class of beneficiaries
+otherBeneficiary.description.error.invalidFormat = The description for the class of beneficiaries must only include letters a to z, numbers, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
+otherBeneficiary.description.error.length = The description for the class of beneficiaries must be 56 characters or less
```

Comment:
Shouldn't refer to class of beneficiaries

Reply:
Fixed now